src/System.Device.Gpio/Interop/Unix/Interop.Library.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    private const string LibcLibrary = "libc";
```

Comment:
NIT: Should this file be located under the libc folder instead?

Reply:
Ok sounds good, looks like we will be adding at least one more since @buyaa-n is working on adding libgpiod support which will require PInvoking as well. One thing then, can we rename the file to be Interop.Libraries.cs instead since the intention is for them to be plural?

Reply:
Gotcha.  Thx for the feedback.